### ISSUE TITLE
My resluts

### ISSUE THREAD

**Issue by EmTee70** [JinShil/memcpyD#1]
I tested with dmd and ldc without further options at Ubuntu 16.4 on an 6 Core AMD Athlon(tm) II X4 640 Processor with 12 GB Ram

[output.txt](https://github.com/JinShil/memcpyD/files/2097143/output.txt)

**Comment by EmTee70** [comment]
I a ask you to close this issue, but what about putting some more diagrams based on the new data collected for you on the memcpyD/README.md page?

**Comment by JinShil** [comment]
The reason I asked for data from others was so I could see if there were any differences between architectures and platforms that I needed to account for.  From your results and the results of others, I see the results are generally consistent, and my tests weren't just a fluke.  That's good.  

I've now changed the algorithm, and begun expanding on it, but that kindof invalidated the data submitted so far.  If you'd like to run another benchmark with the latest code in the repository, that'd be cool,  but as I change the code, I want to keep these graphs up to date, and I can't be asking you or others to send me new data every few weeks, so I'm only publishing the data that I can create myself on demand.
